src/routes/cart.router.js

```diff
@@ -0,0 +1,22 @@
+const express = require("express");
+const cartController = require("../controllers/cart.controller");
+const loginRequired = require("../utils/loginRequired");
+const cartRouter = express.Router();
+
+cartRouter.post("/putItem", loginRequired.verifyUser, cartController.putItem);
```

Comment:
1. RESTful API는 엔드포인트에 동사 사용은 하지 않습니다.
2. loginRequired.verifyUser 대신 verifyUser 함수를 바로 사용할 수 있도록 해당 메서드를 import 하면 좋을 것 같습니다.
3. putItem() 이라는 함수 이름이 애매한 것 같습니다! `createCart()`등 장바구니를 생성하는 의미가 내포될 수 있도록 수정해주세요!

```suggestion
cartRouter.post("/", verifyUser, cartController.createCart);
```